shared/general/labels.md

```diff
@@ -6,6 +6,7 @@ io.{{ $names.company.short }}.features.sysfs | false | Bind mounts the host OS `
 io.{{ $names.company.short }}.features.procfs | false | Bind mounts the host OS `/proc` into the container. | v10.8.0 | v2.48.0
 io.{{ $names.company.short }}.features.kernel-modules | false | Bind mounts the host OS `/lib/modules` into the container (i.e. `/lib/modules:/lib/modules`). | v7.23.0 | v2.21.0
 io.{{ $names.company.short }}.features.firmware | false | Bind mounts the host OS `/lib/firmware` into the container. | v7.23.0 | v2.21.0
+io.{{ $names.company.short }}.features.journal-logs | false | Bind mounts journal log directories `/var/log/journal` and `/run/log/journal` aswell as `/etc/machine-id` in read only mode. Required by some logging agents such as `promtail`. Journal logs can be read using libraries such as `sd-journal` in C or `sdjournal` in Go. | vXX.XX | vXX.XX
```

Comment:
```suggestion
io.{{ $names.company.short }}.features.journal-logs | false | Bind mounts journal log directories `/var/log/journal` and `/run/log/journal` as well as `/etc/machine-id` in read only mode. Required by some logging agents such as `promtail`. Journal logs can be read using libraries such as `sd-journal` in C or `sdjournal` in Go. | vXX.XX | vXX.XX
```

Reply:
also, should those `xx.xx` be replaced with actual versions?

Reply:
thanks for the review @xginn8, yep i will update the versions when `meta-balena` has a new version including a new `balena-supervisor`